bottles/frontend/ui/details-bottle.blp

```diff
@@ -148,78 +115,151 @@ template DetailsBottle : .AdwPreferencesPage {
 
-        Label label_name {
-          halign: start;
-          label: _("My bottle");
+        Label label_arch {
+          label: _("Win64");
 
           styles [
-            "title-2",
+            "heading",
           ]
         }
 
-        Box {
-          spacing: 6;
+        Label {
+          label: "·";
+        }
 
-          Label label_arch {
-            halign: start;
-            label: _("Win64");
+        Label label_environment {
+          label: _("Environment");
 
-            styles [
-              "heading",
-            ]
-          }
+          styles [
+            "heading",
+          ]
+        }
 
-          Label label_environment {
-            label: _("Environment");
+        Label {
+          label: "ꞏ";
+        }
 
-            styles [
-              "heading",
-            ]
-          }
+        Label label_runner {
+          label: _("Runner");
+
+          styles [
+            "heading",
+          ]
+        }
 
-          Label label_runner {
-            label: _("Runner");
+        Label dot_versioning {
+          label: "ꞏ";
+        }
 
-            styles [
-              "heading",
-            ]
-          }
+        Grid grid_versioning {
+          tooltip-text: _("Versioning enabled for this bottle");
+          halign: center;
+          valign: center;
 
-          Grid grid_versioning {
-            tooltip-text: _("Versioning enabled for this bottle");
+          Image img_versioning {
+            tooltip-text: _("Versioning is active for this bottle.");
             halign: center;
             valign: center;
+            icon-name: "view-wrapped-symbolic";
+          }
 
-            Image img_versioning {
-              tooltip-text: _("Versioning is active for this bottle.");
-              halign: center;
-              valign: center;
-              icon-name: "view-wrapped-symbolic";
-            }
-
-            Label label_state {
-              halign: center;
-              valign: center;
-              label: _("0");
-            }
+          Label label_state {
+            halign: center;
+            valign: center;
+            label: _("0");
           }
         }
       }
+    }
+  }
 
-      .AdwSplitButton btn_execute {
-        label: _("Run Executable…");
-        hexpand: true;
-        halign: "end";
-        valign: "center";
-        popover: "pop_run";
+  .AdwPreferencesGroup {
+    Box {
+      spacing: 6;
+      halign: center;
+
+      Button btn_execute {
+        Box {
+          spacing: 6;
+          Image {
+            icon-name: "media-playback-start-symbolic";
+          }
+
+          Label {
+            label: _("Run Executable…");
+          }
+        }
 
         styles [
+          "pill",
           "suggested-action",
         ]
       }
+
+      Box {
+        Popover btn_exec_settings {
+          width-request: 300;
+          
+          Box {
+            orientation: vertical;
+            margin-top: 12;
```

Comment:
```suggestion
            margin-top: 6;
```

Reply:
Initially it used to be 6, but that seemed less to me.

Reply:
f34c63e9eff497e6c1d9a46882456f528fbf8895